### ISSUE TITLE
[L4D2] More colors

### ISSUE THREAD

**Issue by dragokas** [ValveSoftware/Source-1-Games#3469]
Add, please, the ability to use more colors of text in chat.
At the moment, it is only 6 colors available.

**Comment by MartLegion** [comment]
I agree even native colors like red, blue and white we need a player on team to be able to output these colors to the chat. (also can only output one of these at same time)
